src/Commands/Backup/Automatic/InfoCommand.php

```diff
@@ -30,9 +30,9 @@ class GetCommand extends TerminusCommand implements SiteAwareInterface
      *
      * @param string $site_env Site & environment to get the schedule of, in the format `site-name.env`.
      *
-     * @usage terminus backup:schedule:get <site>.<env>
+     * @usage terminus backup:automatic:info <site>.<env>
      *     Responds with the day of the week backups are scheduled for on the <env> environment of <site>
-     * @usage terminus backup:schedule:get awesome-site.dev --format=table
+     * @usage terminus backup:automatic:info awesome-site.dev --format=table
```

Comment:
Should be:
`  * @usage terminus backup:automatic:info <site>.<env> --format=table`